packages/core/config/webpack.common.js

```diff
@@ -250,12 +250,7 @@ module.exports = (webpackEnv) => {
                 },
               },
             },
-            // (2)
-            {
-              test: /\.svg$/,
-              loader: 'svg-react-loader',
```

Comment:
This package should be removed from `core/package.json`

Reply:
Done